### ISSUE TITLE
Make it possible to copy/mv/etc. between two panes.

### ISSUE THREAD

**Issue by ripxorip** [philip-karlsson/bolt.nvim#2]
These features are requested in order to have any use of the dual pane mode. Currently the commands, BoltCopy etc. only supports the dest as argument. One idea is to create additional commands e.g. BoltCopyDual to automatically get the dest from the neighboring pane.

**Comment by johanssondavid** [comment]
At the same time re-map the keys to have Ctrl+# (# = 1,2,3...) as the same functionallity as F# in TC

**Comment by ripxorip** [comment]
@johanssondavid Agree, in that way we will avoid ackward mappings and stay somewhat compatible with TC mappings.

**Comment by johanssondavid** [comment]
It is not possible to map Ctrl-# in vim. https://groups.google.com/forum/#!topic/vim_use/iszb6GfvkRU

**Comment by ripxorip** [comment]
@johanssondavid then I think we shall go for the F-buttons as in TC.

**Comment by ripxorip** [comment]
Also, it would be nice to be able to make selection of files with <SPC> (space)
